src/App.js

```diff
@@ -1,21 +1,55 @@
 import "./App.css";
-import { Routes, Route } from "react-router-dom";
+import { Routes, Route, Navigate } from "react-router-dom";
 import LoginPage from "./pages/LoginPage";
 import RegisterPage from "./pages/RegisterPage";
 import DashboardPage from "./pages/DashboardPage";
 import OrdersPage from "./pages/OrdersPage";
 import ProductsPage from "./pages/ProductsPage";
 import UsersPage from "./pages/UsersPage";
+import Navbar from "./components/Navbar";
+import Footer from "./components/Footer";
+import Sidebar from "./components/Sidebar";
+
 function App() {
+  const isLoggedIn = true;
+  // localStorage.getItem("isLoggedIn") === "true";
+
+  const Layout = ({ children }) => (
```

Comment:
why do you make layout and auto layout we cam make them in one component like authpovider and inside it if login show the sidebar rather than not show sidebar